### PR TITLE
Extract location data from Json, and display them on map

### PR DESCRIPTION
**Pull request by ElementMo**

### CODE REVIEW

File: load-json.js [hunk 1]
```diff
@@ -2,26 +2,45 @@ var loadedJSON = null;
...
-            catch (e)
-            {
+            catch (e) {
                 alert("Unable to parse JSON");
             }
             console.log(loadedJSON);
+
+            // create loc data array
+            var locationData = [];
+            for(var i=0; i<loadedJSON.locations.length; i++){
```
Comment (EgMan) [inline load-json.js]: This should fail with an alert message if there is no "locations" array or if the "locations" array is not of the correct format.

File: load-json.js [hunk 2]
```diff
@@ -2,26 +2,45 @@ var loadedJSON = null;
...
+                locationData.push([loadedJSON.locations[i].longitudeE7 * 0.0000001, loadedJSON.locations[i].latitudeE7 * 0.0000001]);
+            }
+            // create new option
+            newOption = {
+                series: [
+                    {
+                        // data: [
```
Comment (EgMan) [inline load-json.js]: remove unneeded code

File: load-json.js [hunk 3]
```diff
@@ -2,26 +2,45 @@ var loadedJSON = null;
...
+                        //     [-77.04579, 38.89695],
+                        //     [-77.045, 38.896]
+                        // ]
+                        data: locationData
+                    }
+                ]
+            }
+            // update option
+            echartslayer.chart.setOption(newOption);
```
Comment (EgMan) [inline load-json.js]: Is it possible to dynamically/periodically load these points to the map as we parse through the json object? This way, we don't have to see just a loading screen, we could have the points populate as we process them.

Comment (EgMan) [inline load-json.js]: Nice, this works, I tried it.  That being said, the points disappear and re-render every time you move the map and it's quite slow.  Is there any way around this?

File: load-map.js [hunk 4]
```diff
@@ -1,3 +1,5 @@
+var echartslayer;
+
 var map = new mapboxgl.Map(
```
Comment (EgMan) [inline load-map.js:3]: After playing around with my location map for a bit, I realized that we should maybe display city names or something so it's easy to have bearings on where we are.  If this is possible, we can make this a future enhancement.

No resolution needed.

File: load-json.js (hunk 3 above)
Comment (ElementMo) [reply load-json.js]: There is a way to do this. It requires echarts-gl extention, I'll update this later.

File: load-json.js (hunk 2 above)
Comment (ElementMo) [reply load-json.js]: Deleted.

File: load-json.js (hunk 1 above)
Comment (ElementMo) [reply load-json.js]: Fixed.

File: load-map.js (hunk 4 above)
Comment (ElementMo) [reply load-map.js:3]: Okay, I'll dig into that.

Comment (EgMan) [reply load-map.js:3]: +1

File: load-json.js (hunk 3 above)
Comment (EgMan) [reply load-json.js]: +1

Comment (EgMan) [reply load-json.js]: future enhancement

**Review by WayneLee97 — APPROVED** [review]

**Review by user833 — APPROVED** [review]

**Review by EgMan — APPROVED** [review]

**Pull request merged**